### PR TITLE
K8SPXC-372 use new cert-manager API Group

### PR DESCRIPTION
**Pull request by delgod**
[K8SPXC-372](https://jira.percona.com/browse/K8SPXC-372)

since 0.11 version cert-manager has the new API Group

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| recreate | passed |
| big-data | passed |
| init-deploy | passed |
| scaling | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| limits | passed |
| upgrade | passed |
| monitoring | passed |
| restore-to-encrypted-cluster | failed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| monitoring-2-0 | passed |
| upgrade-consistency | passed |
| security-context | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| demand-backup | failed |

File: e2e-tests/functions [hunk 1]
```diff
@@ -366,7 +366,7 @@ run_pumba() {
 deploy_cert_manager() {
     kubectl_bin create namespace cert-manager || :
     kubectl_bin label namespace cert-manager certmanager.k8s.io/disable-validation=true || :
-    kubectl_bin apply -f https://github.com/jetstack/cert-manager/releases/download/v0.14.1/cert-manager.yaml --validate=false || : 2>/dev/null
+    kubectl_bin apply -f https://github.com/jetstack/cert-manager/releases/download/v0.15.1/cert-manager.yaml --validate=false || : 2>/dev/null
 }
 
 destroy() {
```
Comment (boTux) [inline e2e-tests/functions:372]: You might need to upgrade the destroy function in the tests to version v0.15.1 too.

Comment (delgod) [reply e2e-tests/functions:375]: Hi @boTux 

thank you for your feedback, I have fixed deletion of cert-manager and added check if `Issuer` and `Certificates` was created by cert-manager correctly

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| init-deploy | passed |
| big-data | passed |
| storage | passed |
| scaling | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| limits | passed |
| restore-to-encrypted-cluster | passed |
| monitoring | passed |
| upgrade | passed |
| self-healing-advanced | passed |
| monitoring-2-0 | passed |
| upgrade-consistency | passed |
| security-context | passed |
| operator-self-healing | passed |
| demand-backup | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| scheduled-backup | failed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| demand-backup-encrypted-with-tls | failed |

**Pull request merged**